source/beamline/BioXAS/BioXASUtilitiesGroup.cpp

```diff
@@ -0,0 +1,88 @@
+#include "BioXASUtilitiesGroup.h"
+
+BioXASUtilitiesGroup::BioXASUtilitiesGroup(const QString &name, QObject *parent) :
+	BioXASBiStateGroup(name, parent)
+{
+	// Setup value options.
+
+	addOption(Bad, "Bad", true);
+	addOption(Good, "Good", true);
+}
+
+BioXASUtilitiesGroup::~BioXASUtilitiesGroup()
+{
+
+}
+
+bool BioXASUtilitiesGroup::isBad() const
```

Comment:
Happiness is not an option for utilities on BioXAS. It's a black and white world over here!